charts/spark-umbrella/README.md

```diff
@@ -37,11 +37,9 @@ Once everything is up and running you will see something like this:
 ```text
 NAME                              TYPE           CLUSTER-IP      EXTERNAL-IP     PORT(S)           AGE
 kubernetes                        ClusterIP      10.63.240.1     <none>          443/TCP           1d
-spark-all-jupyter-notebook-ui     LoadBalancer   10.63.246.130   35.184.71.164   8888:31540/TCP    9m
-spark-all-jupyter-spark-web-ui    LoadBalancer   10.63.246.140   23.251.149.136  4040:31810/TCP    9m
+spark-all-jupyter-spark           LoadBalancer   10.63.246.130   35.184.71.164   8888:31540/TCP    9m
 spark-all-rss                     LoadBalancer   10.63.246.190   35.192.235.35   10000:31000/TCP   9m
 spark-all-zeppelin                LoadBalancer   10.63.254.150   35.192.68.147   8080:30522/TCP    9m
-spark-all-zeppelin-spark-web-ui   LoadBalancer   10.63.253.49    35.188.92.84    4040:32276/TCP    9m
```

Comment:
For spark-all-jupyter-spark and spark-all-zeppelin services, the PORTS column will also include 4040 port

Reply:
Done.